### DISCUSSION TITLE
Welcome to FearlessTech.github.io Discussions!

### DISCUSSION THREAD

**Discussion by 4-Leafs-Code** [FearlessTech/FearlessTech.github.io#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ProCO-RPHopkins** [comment]
Hi Clover,
I'm currently incarcerated in Maine and would be interested to learn more about Fearless Tech and your work with reentry. I'm working on a project, which you can listen to this podcast to learn more: [App Podcast](https://drive.google.com/file/d/1mT-4KxO_uOgaJxdhBGLJC1iXkEx7RCxR/view?usp=sharing)

If you're interested in having a conversation and potentially collaborating, please shoot me an email: ryan.hopkins2@snhu.edu
